lib/bitrise_conn.py

```diff
@@ -87,73 +32,44 @@ def get_apps(self):
     def set_app(self, project, apps):
         if apps is not None:
             if project == "android":
-                self.APP_SLUG = apps['data'][0]['slug']
+                self.BITRISE_APP_SLUG = apps['data'][0]['slug']
             elif project == "ios":
-                self.APP_SLUG = apps['data'][1]['slug']
+                self.BITRISE_APP_SLUG = apps['data'][1]['slug']
 
     def get_app(self, project, apps):
-        if not self.APP_SLUG:
-            resp = requests.get('https://api.bitrise.io/v0.1/apps/{0}'.
-                                format(self.APP_SLUG),
+        url = self.__url
+        if not self.BITRISE_APP_SLUG:
+            resp = requests.get('{0}''{1}'.
+                                format(url, self.BITRISE_APP_SLUG),
                                 headers=self.API_HEADER)
             if resp.status_code != 200:
                 raise _logger.error('GET /apps/ {}'.format(resp.status_code))
             return resp.json()
 
-    def get_workflows(self, APP_SLUG):
+    def get_workflows(self, BITRISE_APP_SLUG):
+        url = self.__url
         resp = \
-            requests.get('https://api.bitrise.io/v0.1/apps/{0}'
-                         '/build-workflows'.format(self.APP_SLUG),
+            requests.get('{0}{1}'
+                         '/build-workflows'.format(url, self.BITRISE_APP_SLUG),
                          headers=self.API_HEADER)
         if resp.status_code != 200:
             raise _logger.error('GET /apps/ {}'.format(resp.status_code))
         return resp.json()
 
-    def get_builds(self, params=None):
+    def get_builds(self, BITRISE_APP_SLUG):
+        url = self.__url
+        days_ago = 1
+        today = datetime.datetime.utcnow().date()
+        past_date = today - datetime.timedelta(days=days_ago)
```

Comment:
I would need to think how to narrow this down so that we are sure that we count the builds after sometime and right before the github action runs.